src/test/java/org/opensearch/searchrelevance/action/judgment/JudgmentsIT.java

```diff
@@ -120,4 +179,46 @@ public void testMainActions_whenImportReadJudgments_thenSuccessful() {
             )
         );
     }
+
+    @SneakyThrows
+    public void testListJudgments_filteringByStatus_thenSuccessful() {
+        String createBase = """
+            {
+              "name": "FilterTest",
+              "type": "IMPORT_JUDGMENT",
+              "judgmentRatings": [
+                {
+                  "query": "shoes",
+                  "ratings": [
+                    { "docId": "abc", "rating": "1.0" }
+                  ]
+                }
+              ]
+            }
+            """;
+
+        // Create 4 judgments
+        String completedId = createJudgment(createBase);
+        String processingId = createJudgment(createBase);
+        String errorId = createJudgment(createBase);
+        String timeoutId = createJudgment(createBase);
+
+        Thread.sleep(DEFAULT_INTERVAL_MS);
+
+        // Force statuses
+        updateStatus(completedId, "COMPLETED");
+        updateStatus(processingId, "PROCESSING");
+        updateStatus(errorId, "ERROR");
+        updateStatus(timeoutId, "TIMEOUT");
```

Comment:
for last three statuses, looks like we are using integ test to test all possible variants. Let's utilize unit tests for this, integ test are expensive to run and should be reserved for basic end to end scenarios or as a last resort for critical flows.

Reply:
@martin-gaievski Would this simplified version of the integration test be sufficient?
I removed the tests for the other three statuses and kept only the COMPLETED case.
Let me know if you’d prefer adding separate unit tests to cover PROCESSING, ERROR, and TIMEOUT as well.